DEVELOPMENT_CONTRIBUTION.md

```diff
@@ -129,31 +130,37 @@ As you implement your contribution, make sure all work stays on your local topic
 
 To make sure your pull request will pass our automated testing, before submitting you should:
 
+* Make sure `npm run test:golden` passes.
 * Make sure `npm test` passes for each of them.
-* Make sure `npm run clang:check` passes for each of them. If not, you can run `npm run clang:format` to format your code.
-* Make sure `npm run tslint:check` passes for each of them. If not, you can run `npm run tslint:fix` to try to auto-fix some of them.
   For certain lint failures you will have to fix them manually.
 
-If everything passes, you can push your changes to your fork of Clarity, and [submit a pull request](https://help.github.com/articles/about-pull-requests/). Remember, submit it **against the topic branch** on the main Clarity repository, not `master`!
+To test the same thing that the CI will test you could run `npm run test:travis`
+
+If everything passes, you can push your changes to your fork of Clarity, and [submit a pull request](https://help.github.com/articles/about-pull-requests/).
+
+* Assign yourself to the Pull-Request
+* Assign proper labels for example if you are making documentation update only use `documentation`, `website`
+* Assign connected Issue that this PR will resolve
 
 ### Taking reviews into account
 
-During the review process of your pull request(s), some changes might be requested by Clarity team members. If that happens, add extra commits to your pull request to address these changes. Make sure these new commits are also signed and follow our commit message format.
+During the review process of your pull request(s), some changes might be requested by Clarity team members.
+If that happens, add extra commits to your pull request to address these changes.
+Make sure these new commits are also signed and follow our commit message format.
 
-**Do not amend your commits or squash them**, it will only slow down the review process by forcing reviewers to check every line again. We'll clean up the commit history ourselves when we finally merge your pull request, you don't have to worry about it.
+Please keep an eye on you Pull-Request and try to address the comments if any, as soon as posible.
```

Comment:
Something I've seen recently on a few PRs is confusion about rebasing onto master, and how to squash. Can we write a snippet about that here?